code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -289,20 +289,29 @@
 		return
 
 	if(wisp.loc == src)
-		to_chat(user, "<span class='notice'>You release the wisp. It begins to bob around your head.</span>")
-		icon_state = "lantern"
-		wisp.orbit(user, 20)
-		SSblackbox.record_feedback("tally", "wisp_lantern", 1, "Freed")
+		if(wisp.time < world.time)
```

Comment:
use our cooldown defines

Reply:
Where are these defined? or just a term that I can quickly search in VSCode